Translation/Exporter/ExporterCollector.php

```diff
@@ -25,28 +33,23 @@ public function addExporter($id, ExporterInterface $exporter)
     }
 
     /**
-     * Returns an exporter that support the given format.
-     *
-     * @param string $format
-     * @return ExporterInterface
+     * @param $format
+     * @param $file
+     * @param $translations
+     * @return bool
+     * @throws \RuntimeException
      */
-    public function getByFormat($format)
+    public function export($format, $file, $translations)
```

Comment:
@kuczek are devs really using (or going to use) this service method in their src?